src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/DiaryStoringProviding.swift

```diff
@@ -55,10 +68,21 @@ protocol DiaryStoring {
 	func close()
 }
 
-protocol DiaryProviding {
+protocol DiaryStoringV3 {
```

Comment:
Why can't we just extend the existing protocols?

Reply:
If we would extend the protocol, all ContactDiaryStores (V1,V2,V3) should implement the method. But we want it only be available for V3.

Reply:
Are we using these in parallel? If yes, why? Else, let's merge this and have one protocol that's versioned via git and not copy/paste.